mistletoe/core_tokens.py

```diff
@@ -307,6 +307,12 @@ def match_link_label(string, offset, root=None):
 
 
 def is_link_label(text, root):
+    """
+    Normalize and look up `text` among the footnotes.
```

Comment:
Nice one, there is never enough of documentation.

I would also suggest to rename this old method to `get_link_label` - that would be more appropriate, given what the function actually does (and seemingly did)?

Reply:
agree

Reply:
thanks for the rename :)